autoscheduler/frontend/src/redux/reducers/courseCards.ts

```diff
@@ -66,6 +67,7 @@ export default function courseCards(
       return {
         ...state,
         [action.index]: { ...state[action.index], ...action.courseCard },
+        numCardsCreated: Math.max(state.numCardsCreated, action.index + 1),
```

Comment:
I used `updateCourseCardSync` when creating cards from serialized save cards since `addCourseCard` seems a bit unintuitive to use when updating the first card rather than adding it, but the same could be said for calling `updateCourseCard` for cards that don't exist. We can use `addCourseCard` instead here without changing any code, so I could change it to that if you'd prefer (and remove this line since it would no longer be needed)

Reply:
No, updateSync is fine